app/models/debts/debt.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Debts
+  class Debt
+    def initialize(instance)
```

Comment:
@dissident `@instance` is a bad name for variable. What does it mean? Instance of that does it handle?